developers/extensionservices/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: developersguide
+title: Extension service
+---
+
+{% include base.html %}
+
+# Extension service
```

Comment:
Is the extension service already merged into the core? I lost track a bit :)

If not, maybe reference the PR / discussion here and mark this page as "future improvement"?

Reply:
It was introduced for the eclipse marketplace. It needs redoing, but it currently is part of openHAB.

Reply:
As mentioned somewhere else, I would suggest to remove this here as I don't think that the community should start writing extension services. This interface had only been introduced in ESH in order to be able to have an openHAB specific implementation.